localstack/services/cloudwatch/cloudwatch_database_helper.py

```diff
@@ -317,16 +324,25 @@ def list_metrics(self, account_id, region, namespace, metric_name, dimensions) -
 
             query = f"""
                 SELECT DISTINCT metric_name, namespace, dimensions
-                FROM {self.TABLE_SINGLE_METRICS}
-                WHERE account_id = ? AND region = ? {namespace_filter} {metric_name_filter} {dimension_filter}
+                FROM (
+                    SELECT metric_name, namespace, dimensions, account_id, region, timestamp
+                    FROM SINGLE_METRICS
+                    UNION
+                    SELECT metric_name, namespace, dimensions, account_id, region, timestamp
+                    FROM AGGREGATED_METRICS
+                ) AS combined
+                WHERE account_id = ? AND region = ?
+                {namespace_filter}
+                {metric_name_filter}
+                {dimension_filter}
                 ORDER BY timestamp DESC
```

Comment:
nice 🎉 
Let's see how we do with the performance in the future. We may be able to improve the query, if we add the `WHERE` clause to the two inner selects already :)